test/contrib/elasticsearch/integration_test.rb

```diff
@@ -0,0 +1,30 @@
+require 'time'
+require 'contrib/elasticsearch/test_helper'
+require 'helper'
+
+class ESIntegrationTest < Minitest::Test
+  ELASTICSEARCH_SERVER = 'http://127.0.0.1:49200'.freeze
+  def setup
+    skip unless ENV['TEST_DATADOG_INTEGRATION'] # requires a running agent
+
+    # Here we use the default tracer (to make a real integration test)
+    @tracer = Datadog.tracer
```

Comment:
can we use a copy of a Tracer (i.e. creating a new Tracer with defaults)? Because actually I always created a copy (or change that if I didn't 😄 ) since we may change the behavior of a tracer in some tests. In this way we're sure that each test uses it's own tracer that can be changed at runtime.